template/styles/css/main.css

```diff
@@ -832,15 +832,14 @@ a.disable:hover {
   min-height: 100%; }
 
 .body-content {
-  display: -webkit-box;
```

Comment:
Is there a reason these styles were moved?

Reply:
The main.css is the compiled sass dir, this is removed because of the removal of the flex layout in the body-content